### PR TITLE
Brazilian Portuguese translation of html5-css challenges.

### LINKED ISSUE #6708: Brazilian Portuguese translation of HTML5 and CSS
- [ ] Say Hello to HTML Elements
- [ ]  Headline with the h2 Element
- [ ]  Inform with the Paragraph Element
- [ ] Uncomment HTML
- [ ] Comment out HTML
- [ ]  Fill in the Blank with Placeholder Text
- [ ]  Delete HTML Elements
- [ ]  Change the Color of Text
- [ ]  Use CSS Selectors to Style Elements
- [ ]  Use a CSS Class to Style an Element
- [ ]  Style Multiple Elements with a CSS Class
- [ ]  Change the Font Size of an Element
- [ ]  Set the Font Family of an Element
- [ ]  Import a Google Font
- [ ]  Specify How Fonts Should Degrade
- [ ]  Add Images to your Website
- [ ]  Size your Images
- [ ]  Add Borders Around your Elements
- [ ]  Add Rounded Corners with a Border Radius
- [ ]  Make Circular Images with a Border Radius
- [ ]  Link to External Pages with Anchor Elements
- [ ]  Nest an Anchor Element within a Paragraph
- [ ]  Make Dead Links using the Hash Symbol
- [ ]  Turn an Image into a Link
- [ ]  Add Alt Text to an Image for Accessibility
- [ ]  Create a Bulleted Unordered List
- [ ]  Create an Ordered List
- [ ]  Create a Text Field
- [ ]  Add Placeholder Text to a Text Field
- [ ]  Create a Form Element
- [ ]  Add a Submit Button to a Form
- [ ]  Use HTML5 to Require a Field
- [ ]  Create a Set of Radio Buttons
- [ ]  Create a Set of Checkboxes
- [ ]  Check Radio Buttons and Checkboxes by Default
- [ ]  Nest Many Elements within a Single Div Element
- [ ]  Give a Background Color to a Div Element
- [ ]  Set the ID of an Element
- [ ]  Use an ID Attribute to Style an Element
- [ ]  Adjusting the Padding of an Element
- [ ]  Adjust the Margin of an Element
- [ ]  Add a Negative Margin to an Element
- [ ]  Add Different Padding to Each Side of an Element
- [ ]  Add Different Margins to Each Side of an Element
- [ ]  Use Clockwise Notation to Specify the Padding of an Element
- [ ]  Use Clockwise Notation to Specify the Margin of an Element
- [ ]  Style the HTML Body Element
- [ ]  Inherit Styles from the Body Element
- [ ]  Prioritize One Style Over Another
- [ ]  Override Styles in Subsequent CSS
- [ ]  Override Class Declarations by Styling ID Attributes
- [ ]  Override Class Declarations with Inline Styles
- [ ]  Override All Other Styles by using Important
- [ ]  Use Hex Code for Specific Colors
- [ ]  Use Hex Code to Color Elements White
- [ ]  Use Hex Code to Color Elements Red
- [ ]  Use Hex Code to Color Elements Green
- [ ]  Use Hex Code to Color Elements Blue
- [ ]  Use Hex Code to Mix Colors
- [ ]  Use Hex Code to Color Elements Gray
- [ ]  Use Hex Code for Specific Shades of Gray
- [ ]  Use Abbreviated Hex Code
- [ ]  Use RGB values to Color Elements
- [ ]  Use RGB to Color Elements White
- [ ]  Use RGB to Color Elements Red
- [ ]  Use RGB to Color Elements Green
- [ ]  Use RGB to Color Elements Blue
- [ ]  Use RGB to Mix Colors

### PR DESCRIPTION
**Pull request by carolinaknoll**
Closes #6708

### CODE REVIEW

**Comment by carolinaknoll** [comment]
Hello. I noticed there are issues with my PR, and then reviewed my code on JSONLint as recommended, but could not figure them out.    
Tried closing each translation with `]` instead of `],` but then had `"type": "waypoint"` errors, so right now I don't know what to do.    
Can someone please tell me what is wrong so that I can commit the fixes and then squash the commits when everything is correct? Thanks a lot.

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -2336,7 +2653,16 @@
...
+      ],
+     "namePtBR": "Dê uma Cor de Fundo a um Elemento Div",
+     "descriptionPtBR": [
+        "Você pode acrescentar uma cor de fundo de um elemento com a propriedade <code>background-color</code>.",
+        "Por exemplo, se você quiser que a cor de fundo de um elemento seja verde (<code>green</code>), dentro de seu elemento <code>style</code> seria assim:",
+        "<code>.green-background {</code>",
+        "<code>&nbsp;&nbsp;background-color: green;</code>",
+        "<code>}</code>",
+        "Crie uma classe chamada <code>gray-background</code> com a propriedade <code>background-color</code> em cinza (<code>gray</code>). Depois, adicione essa classe ao seu elemento <code>div</code>."
+      ],
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: This should not have a comma after it.

**Comment by ltegman** [comment]
As per https://github.com/FreeCodeCamp/FreeCodeCamp/pull/6930#discussion_r52922538 we do not currently support multiple locales, so `PtB`R should instread be `Pt`.

**Comment by bugron** [comment]
@ltegman :point_up: [February 16, 2016 3:01 AM](https://gitter.im/FreeCodeCamp/CoreTeam?at=56c258e237437b6756097cf5) and the message bellow.

**Comment by carolinaknoll** [comment]
As requested, the commas after the last translation of each challenge were removed. Is that it? Thank you.

**Comment by QuincyLarson** [comment]
@carolinaknoll Thank you so much for your contribution. I manually resolved the conflicts and merged this on my side.

**Comment by bugron** [comment]
@carolinaknoll thanks for your contribution. The only note is that always remember to [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) your multiple commits into one before pushing.

**Pull request merged**